symfony/plugins/orangehrmRecruitmentPlugin/entity/Vacancy.php

```diff
@@ -0,0 +1,247 @@
+<?php
```

Comment:
copyright header & new line end of the file

Reply:
done